src/app/shell/details/competition-details/competition-details.component.ts

```
@@ -6,24 +6,37 @@ import { WINDOW } from 'ngx-window-token';
 import { filter, Observable } from 'rxjs';
 import { take } from 'rxjs/operators';
 
+import { Actions, ofAction, Select, Store } from '@ngxs/store';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Select'. Consider to delete it or rename it.

'Select' is defined here:


(lint/suspicious/noRedeclare)